lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +12,15 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if not isinstance(text, str):
+        return None
+
+    tokens = []
+    text = text.lower()
+    for i in text:
```

Comment:
This is good, but an improvement would be to use a list comprehension. Try it!

Reply:
done